sdk/test-utils/recorder/src/recorder.ts

```diff
@@ -88,35 +91,14 @@ export function record(
     testTitle = testContext.test!.title;
   }
 
-  setEnvironmentOnLoad();
-
-  if (isBrowser()) {
-    recorder = new NiseRecorder(testHierarchy, testTitle);
-  } else {
-    recorder = new NockRecorder(testHierarchy, testTitle);
-  }
-
-  if (isRecordMode()) {
-    // If TEST_MODE=record, invokes the recorder, hits the live-service,
-    // expects that the appropriate environment variables are present
-    recorder.record(recorderEnvironmentSetup);
-  } else if (isPlaybackMode()) {
-    // If TEST_MODE=playback,
-    //  1. sets up the ENV variables
-    //  2. invokes the recorder, play the exisiting test recording.
-    setEnvironmentVariables(env, recorderEnvironmentSetup.replaceableVariables);
-    recorder.playback(recorderEnvironmentSetup, testContext.currentTest!.file!);
-  }
-  // If TEST_MODE=live, hits the live-service and no recordings are generated.
-
-  return {
+  const result: Recorder = {
```

Comment:
I moved this to exist earlier in this constructor so I could return it before having to go through more complex conditionals.